acceptance/config/puppetgit/acceptance-options.rb

```diff
@@ -1,4 +1,9 @@
 {
+    :type                 => 'foss',
+    :add_el_extras        => 'true',
+    :is_puppetserver      => false,
+    :puppetservice        => 'puppet.service',
     :pre_suite            => 'acceptance/pre_suite/puppet_git/install.rb',
-    :tests                => 'acceptance/tests/puppet'
-}.merge(eval File.read('acceptance/config/acceptance-options.rb'))
\ No newline at end of file
+    :tests                => 'acceptance/tests/puppet',
```

Comment:
why are we running all the puppet tests against each puppet install type?  i suppose before we added this one it was how we were testing the install stuff.  feels kinda backwards to me.